pkg/cloud-provider/cloudapi/azure/azure_security.go

```diff
@@ -709,21 +711,24 @@ func (computeCfg *computeServiceConfig) getAGGroupView(nepheControllerAGSgNameTo
 //	SecurityInterface Implementation
 //
 // ////////////////////////////////////////////////////////.
-func (c *azureCloud) CreateSecurityGroup(addressGroupIdentifier *securitygroup.CloudResource, membershipOnly bool) (*string, error) {
+
+// CreateSecurityGroup invokes cloud api and creates the cloud security group based on securityGroupIdentifier.
+// For addressGroup it will attempt to create an asg, for appliedTo groups it will attempt to create both nsg and asg.
```

Comment:
This isnt correct.. For AG it will create ASG and appliedTo will be NSG and Rules.. Not ASG.. Also consider using NSG and ASG in caps

Reply:
For AT it will create both ASG and NSG on Azure